lib/Yam/Agama/Pom/RebootTextmodePage.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles installation reboot screen for textmode display.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::Agama::Pom::RebootTextmodePage;
+use strict;
+use warnings;
+
+use testapi;
+
+sub new {
+    my ($class, $args) = @_;
+    return bless {
+        tag_installation_complete => 'agama-install-finished',
+        tag_reboot_button => 'reboot'
```

Comment:
we don't need this in this one.

Reply:
5df52e594bdbf216d60854b5e3a2e2405a4b14f3